utils/lxc/Makefile

```diff
@@ -25,6 +25,19 @@ PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/meson.mk
+
+TARGET_CFLAGS += -fPIC
+TARGET_LDFLAGS += -fPIC
```

Comment:
meson has -Db_pic=true

Reply:
And actually you probably mean to use -Db_pie.

Reply:
I see this discussion is marked as resolved, but I dont see any change related to it?